report_thesis/src/sections/results/initial_results.tex

```diff
@@ -0,0 +1,50 @@
+\subsection{Initial Results}
+As described in Section~\ref{sec:proposed_approach}, we conducted a series of initial experiments to evaluate the performance of various machine learning models on the prediction of major oxide compositions from our \gls{libs} dataset.
+These experiments aimed to provide a preliminary assessment of the models' performance.
+All models were trained on the same preprocessed data using the Norm 3 preprocessing method described in Section~\ref{sec:norm3}.
+This ensured that the models' performance could be evaluated under consistent and comparable conditions.
+
+Table~\ref{tab:init_results} presents the results of these experiments, including the \gls{rmsep}, \gls{rmsecv}, standard deviation, standard deviation of the cross-validation, and the mean of these metrics for each model and oxide.
+The primary metric of interest is the \gls{rmsecv}, as it provides a measure of the model's predictive accuracy during cross-validation, reflecting its generalization performance on unseen data.
+Additionally, the standard deviation of the cross-validation offers insight into the variability of the model's performance across different folds, and the \gls{rmsep} indicates the model's predictive accuracy on a separate test set.
+
+Ridge regression and \gls{svr} generally exhibit strong performance across different oxides, as evidenced by their relatively low \gls{rmsecv} values.
+In contrast, \gls{enet}, \gls{ann}, and \gls{cnn} tend to perform the worst, with higher \gls{rmsecv} values.
+Models such as \gls{rf} and \gls{gbr} exhibit moderate performance, often surpassing simpler linear models but not as consistently as \gls{svr}.
+
+For $\ce{SiO2}$, \gls{svr} demonstrates the lowest \gls{rmsecv} of 4.908.
+Ridge regression and \gls{rf} also perform well, with \gls{rmsecv} values of 5.005 and 5.304, respectively.
+The standard deviations for these models are relatively low, indicating stable performance.
+
+In the case of $\ce{TiO2}$, \gls{etr} exhibits the lowest \gls{rmsecv} of 0.427, signifying exceptional performance, with \gls{svr} and \gls{ngboost} also showing strong predictive capabilities, with \gls{rmsecv} values of 0.463 and 0.433, respectively.
+The low standard deviation of the cross-validation for these models indicates consistent performance across folds.
+
+For $\ce{Al2O3}$, \gls{ngboost} with an \gls{rmsecv} of 2.291 and ridge regression with an \gls{rmsecv} of 3.115 achieve relatively low \gls{rmsecv}, highlighting their effectiveness in predicting this oxide, with \gls{svr} also performing well with an \gls{rmsecv} of 2.700.
```

Comment:
OK, I'll stop reading at this point, because you consistently highlight the wrong models as having the top performance. I've doubled checked in the data & in the table on those I've already mentioned.

It doesn't make sense to me why we would highlight inferior performance as being good. Please let me know your thoughts, as I'm super confused by this.